package.template.json

```diff
@@ -12,7 +14,8 @@
     "list-dependencies": "node scripts/platform-dependencies",
     "postinstall": "node scripts/postinstall",
     "setup": "cd scripts && yarn install",
-    "start": "react-native start"
+    "start": "react-native start",
+    "link-fonts": "react-native-asset -a extensions/shoutem.fonts/app/fonts/"
```

Comment:
Could you elaborate on when this NPM script is run? Is it a standard hook that's executed during the build or is it something meant to be executed by our build scripts?

Reply:
@Definitely-Not-Vlad this is a workaround to link the fonts on iOS since the default link command does not work anymore, we call it in our customs script after shoutem configuration finished, ideally this is handled by shoutem configuration script